rpmbuild/bin/copr-builder-rhsm-subscribe

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/expect
+
+# Provide username+password for subscription-manager register
+# through RHSM_USER and RHSM_PASS.
+# https://bugzilla.redhat.com/show_bug.cgi?id=2315673
+
+set timeout 30
+set USER $env(RHSM_USER)
+set PASS $env(RHSM_PASS)
```

Comment:
Unfortunately, env vars are not safe either as long as user can ssh to the box:
https://github.com/fedora-copr/copr/blob/35a23e6c655967c499492dabbef10abce2fde57e/backend/copr_backend/background_worker_build.py#L993-L994